python/zenoss/protocols/amqpconfig.py

```diff
@@ -24,13 +60,13 @@ class AMQPConfig(object):
     all of the various queues that zenoss uses.
     """
     _options = [
-        dict(short_opt='H', long_opt='amqphost', type='string', key='host', default='localhost', help='Rabbitmq server host'),
-        dict(short_opt='P', long_opt='amqpport', type='int', key='port', default=5672, help='Rabbitmq server port', parser=int),
-        dict(short_opt='V', long_opt='amqpvhost', type='string', key='vhost', default='/zenoss', help='Rabbitmq server virtual host'),
-        dict(short_opt='u', long_opt='amqpuser', type='string', key='user', default='zenoss', help='User to connect as'),
-        dict(short_opt='p', long_opt='amqppassword', type='string', key='password', default='zenoss', help='Password to connect with'),
-        dict(short_opt='s', long_opt='amqpusessl', action='store_true', key='usessl', default=False, help='Use SSL to connect to the server', parser=lambda v: str(v).lower() in ('true', 'y', 'yes', '1')),
-        dict(short_opt='b', long_opt='amqpheartbeat', type='int', key='amqpconnectionheartbeat', default=300, help='AMQP Connection Heart Beat in Seconds', parser=int),
+        dict(short_opt='H', long_opt='amqphost', type='string', key='host', default=DEFAULT['amqphost'], help='Rabbitmq server host'),
+        dict(short_opt='P', long_opt='amqpport', type='int', key='port', default=DEFAULT['amqpport'], help='Rabbitmq server port', parser=int),
+        dict(short_opt='V', long_opt='amqpvhost', type='string', key='vhost', default=DEFAULT['smqpvhost'], help='Rabbitmq server virtual host'),
+        dict(short_opt='u', long_opt='amqpuser', type='string', key='user', default=DEFAULT['amqpuser'], help='User to connect as'),
+        dict(short_opt='p', long_opt='amqppassword', type='string', key='password', default=DEFAULT['amqppassword'], help='Password to connect with'),
+        dict(short_opt='s', long_opt='amqpusessl', action='store_true', key='usessl', default=DEFAULT['amqpusessl'], help='Use SSL to connect to the server', parser=lambda v: str(v).lower() in ('true', 'y', 'yes', '1')),
+        dict(short_opt='b', long_opt='amqpheartbeat', type='int', key='amqpconnectionheartbeat', default=DEFAULT['amqpheartbeat'], help='AMQP Connection Heart Beat in Seconds', parser=int),
```

Comment:
aamqpheartbeat not in defaults

Reply:
Added the key and its default value to the dictionary.